fnet/nn_modules/fnet_nn_3d.py

```diff
@@ -1,16 +1,15 @@
 import torch
-import pdb
 
 # parent: v5_nn.py
 # changed from v5: removed ReLU at output
 
 class Net(torch.nn.Module):
-    def __init__(self):
+    def __init__(self, out_channels = 1):
         super().__init__()
         mult_chan = 32
         depth = 4
         self.net_recurse = _Net_recurse(n_in_channels=1, mult_chan=mult_chan, depth=depth)
```

Comment:
fwiw i would make all these parameters keyword arguments, that's what i've done.

Reply:
Let's keep the original `fnet_nn_3d.py ` without parameters. We can define a copy of it with all parameters specifiable.